lib/tasks/data/truncate_data.rake

```diff
@@ -6,6 +6,22 @@ require 'tasks/data/truncate_data'
 # This task can be used to significantly reduce the size of a database
 #   This is used for example when loading live data into a staging server
 #   This way the staging server is not overloaded with too much data
+#
+# This is also aimed at implementing data archiving. We assume data older than
+# 2 years can be safely removed and restored from a backup. This gives room for
+# hubs to do their tax declaration.
+#
+# It's a must to perform a backup right before executing this. Then, to allow
+# for a later data recovery we need to keep track of the exact moment this rake
```

Comment:
I wonder if a timestamp is actually written to the logs when executing the task. Maybe we can log it when initializing the TruncateData service?

Reply:
that would improve the situation a bit but the way to keep track of the event IMO is through a Datadog event like we do for deploys and provisions, so it doesn't get lost. Unfortunately, AU is not in Datadog, the 2nd best option we have is sending an event to Slack, I think.

Reply:
datadog, awesome :+1: